src/client/datascience/notebook/vscodeNotebookController.ts

```diff
@@ -172,13 +164,6 @@ export class VSCodeNotebookController implements Disposable {
         if (event.selected) {
             await this.updateCellLanguages(event.notebook);
             this._onNotebookControllerSelected.fire({ notebook: event.notebook, controller: this });
-        } else {
-            // If this controller was what was previously selected, then wipe that information out.
-            // This happens when user selects our controller & then selects another controller e.g. (.NET Extension).
-            // If the user selects one of our controllers (kernels), then this gets initialized elsewhere.
-            if (isSameAsTrackedKernelInNotebookMetadata(event.notebook, this.connection)) {
```

Comment:
DOn't wipe it, we could end up wiping data setup by other controllers (e.g. by .NET controller or the like).